tests/unit_tests/dft/include/compute_inplace.hpp

```diff
@@ -165,6 +163,8 @@ int DFT_Test<precision, domain>::test_in_place_buffer() {
         }
     }
 
+    // account for scaling that occurs during DFT
+    std::for_each(input.begin(), input.end(), [this](auto& x) { x *= forward_elements; });
```

Comment:
Why did you need to do that instead of using the backward scale as it was before?

Reply:
So if a user of the oneMKL Interface library is using the oneMKL SYCL APIs with cuFFT backend and setting the bwd_scale then the scale will not be applied to the result of the FFT?

Reply:
Yes, that does seem bad.
I could add a check to the commit step so that an exception is thrown when `BACKWARD|FORWARD_SCALE != 1`

Reply:
I've added a fix in https://github.com/oneapi-src/oneMKL/pull/284/commits/fb06c4621a0555f0a2facad396e7603ba3a0466e. This does have a problem in that I had to rearrange the test since the invalid scale was stored, and then all subsequent attempts to commit became invalid.

Reply:
I think we should find a way to make the backward, forward scale works for the cuFFT backend.
The goal of this project is to demonstrate that we can implement the oneMKL SYCL APIs as defined in the oneMKL specification for many platforms.
So as is we have a gap for Nvidia platform.
We could add the scale for the Nvidia backend in a different PR though and as a first step maybe throw an "unimplemented" exception for those scale for the cuFFT backend and update the test only for the cuFFT backend to either apply the scale on the test side if the exception is catch or just simply skip scaling in the cuFFT backend tests.

Reply:
I think the team is open to adding an extra kernel that will do the scaling when `FORWARD/BACKWARD_SCALE != 1` if it comes with some documentation that for cuFFT it will likely add overhead.
When we get to adding the FORWARD/BACKWARD_SCALE tests, a range of values will be tested and the test will skip when a backend throws an "unimplemented" exception.